src/components/React/ColorMappingController.js

```diff
@@ -1,73 +1,31 @@
 import React, { Component } from "react";
 import ReactModal from "react-modal";
+// import { DataTable } from "primereact/datatable";
+// import { Column } from "primereact/column";
+import "primereact/resources/themes/nova/theme.css";
+import "primereact/resources/primereact.min.css";
+import "primeicons/primeicons.css";
+
 import { connect } from "react-redux";
 import {
   myChangeColorMapAction,
```

Comment:
don't prefix these with `my`

Reply:
I'll change the imports in `ColorMapping.js` for now

Variable/ function name changes will be a separate PR. After this I'll move controls to context - issue #38